### ISSUE TITLE
sql: Make it possible to rename tables and columns underlying a view

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#1573]
Background: https://airtable.com/tblD3oZPLJgGhCmch/viwdV1sM5KEK2L14K/rechvwHLkLhbmWisX

PRs:
- https://github.com/cockroachdb/cockroach/pull/15388

**Comment by jseldess** [comment]
@knz, once this is ready for docs, please apply the `ready` label and add links to relevant PRs. At the minimum, we'll have to remove notes like at the top of https://www.cockroachlabs.com/docs/rename-column.html.

**Comment by jseldess** [comment]
@knz, just saw https://github.com/cockroachdb/cockroach/pull/16884. Is that part of this larger project? If so, is the larger project ready to document?

**Comment by knz** [comment]
Not yet, no.

**Comment by knz** [comment]
However I just noticed that #16884 needs documentation of its own. I added the tags on it.

**Comment by jseldess** [comment]
Hey Raphael. Did this make it into 1.1? Is it ready to be documented?

**Comment by knz** [comment]
Not yet. still working on it.

**Comment by jseldess** [comment]
Pinging you again, @knz. Is this getting punted to 1.2 at this point?

**Comment by knz** [comment]
unfortunately, yes.

**Comment by jseldess** [comment]
OK, thanks for the update, @knz.

**Comment by jseldess** [comment]
Closing in favor of https://github.com/cockroachdb/docs/issues/2018.
